projects/smart-table/src/lib/smart-table.types.ts

```diff
@@ -10,7 +10,8 @@ export interface IModuleConfig {
   options?: {
     useLowerCaseQueryParams?: boolean,
     noConfigApiCall?: boolean,
-    noExport?: boolean
+    noExport?: boolean,
+    exportWithFilters?: boolean
```

Comment:
For next time … you can suffix every line with a comma, which makes PRs more readable.

Reply:
We could enable `comma-dangle` in the config?

Reply:
toegevoegd en aangepast